apollo-ios/Sources/Apollo/ExecutionSources/CacheDataExecutionSource.swift

```diff
@@ -55,7 +59,7 @@ struct CacheDataExecutionSource: GraphQLExecutionSource {
                 } else {
                   return error
                 }
-              }
+              }.map { $0 as AnyHashable }
           }.map { $0._asAnyHashable }
```

Comment:
It's not ideal that we are having to map this and cast is at `AnyHashable` twice here, but it might be unavoidable. Are you certain we need these both?

Reply:
I believe it's only the result type from `mapError` that actually needs to be cast; it's a build error without it. So there shouldn't actually be any casting unless there was an error no?